terraform/environments/main.tf

```diff
@@ -128,7 +128,7 @@ module "aws_deploy-ap-southeast-1" {
   ami_name      = "aeternity-ubuntu-16.04-v1549009934"
 
   aeternity = {
-    package = "https://releases.ops.aeternity.com/aeternity-2.4.0-ubuntu-x86_64.tar.gz"
+    package = "https://releases.ops.aeternity.com/aeternity-3.0.0-rc.1-ubuntu-x86_64.tar.gz"
```

Comment:
For all items with `3.0*`, please comment on bootstrap_version: I see v2.0.1 though maybe you meant [v2.2.0](https://github.com/aeternity/infrastructure/compare/v2.0.1...v2.2.0).

Reply:
good point, I updated all environments bootstrap version to 2.2.0 in 2612809